atest/acceptance/keywords/print_page.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Documentation    Suite description
+Suite Setup       Go To Page "non_ascii.html"
+Resource          ../resource.robot
+
+Test Setup    Remove Files    ${OUTPUTDIR}/selenium-page-*.pdf
+
+*** Test Cases ***
+Print Page As PDF Without Print Options

```

Comment:
Is this same as next test?

Reply:
I guess it is but not not intended to be. The first was intended to make sure that a page would print if no arguments were passed to it and iusing some of the default settings.

The second one is to check the automatic indexing of the saved filename. But you are right that a single test would check both of these scenarios.

Reply:
Updated the second test using various arguments to make it sloightly different than the first.